### ISSUE TITLE
Persist records

### ISSUE THREAD

**Issue by Stebalien** [libp2p/go-libp2p-pubsub-router#18]
This router doesn't currently persist the records it finds to disk. We should consider doing that but this *may* depend on transaction support.

**Comment by magik6k** [comment]
I don't think this is a good idea. This router returns last 'heard' value immediately as it's fairly safe to assume that it's also what the owner of a given key published most recently.

If it was to persist records, it would need to somehow make sure that no new records were published since, which is something that should be handled in higher layers which do caching/handle TTL/etc.
